Improve HA install steps (#960)
rancher/docs: content/rancher/v2.x/en/faq/technical/_index.md

### How can I add additional arguments/binds/environment variables to Kubernetes components in a Rancher Launched Kubernetes cluster?

You can add additional arguments/binds/environment variables via the [Config File]({{< baseurl >}}/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/#config-file) option in Cluster Options. For more information, see the [Extra Args, Extra Binds, and Extra Environment Variables]({{< baseurl >}}/rke/v0.1.x/en/config-options/services/services-extras/) in the RKE documentation or browse the [Example Cluster.ymls]({{< baseurl >}}/rke/v0.1.x/en/example-yamls/).

> ### How to check `Common Name` and `Subject Alternative Names` in your server certificate?

Reviewer: ```suggestion
### How do I check `Common Name` and `Subject Alternative Names` in my server certificate?
```
